src/components/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+ import {useParams} from 'react-router-dom'; 
+import { useState, useEffect } from 'react';
+import {getArticles } from '../utils/api';
+import axios from 'axios';
+import '../App.css';
+
+
+const Articles = () => { 
+  const [articles, setArticles] = useState([]); 
+  
+    useEffect(() => {
```

Comment:
Should add a loading pattern so the user isn't looking at a blank page while the articles are loading. If you're not sure how to do this check out the last part of the useEffect notes 🙂 https://notes.northcoders.com/courses/js-front-end/useEffect